source/HailPanel.cpp

```diff
@@ -62,8 +62,16 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	else if(gov->IsEnemy() && !ship->IsDisabled())
-		SetBribe(gov->GetBribeFraction());
+	else if(gov->IsEnemy())
+	{
+		// Enemy ships always show hostile messages.
+		// They either show bribing messages,
+		// or standard hostile messages, if disabled.
+		if(ship->IsDisabled())
+			message = ship->GetHail(player.GetSubstitutions());
+		else
+			SetBribe(gov->GetBribeFraction());
```

Comment:
`message = ship->GetHail(player.GetSubstitutions());` is run after this if block if `message` is empty, so don't run the same line here.
```suggestion
		if(!ship->IsDisabled())
			SetBribe(gov->GetBribeFraction());
```